src/i965_encoder_utils.c

```diff
@@ -82,6 +82,7 @@ struct __avc_bitstream {
 
 typedef struct __avc_bitstream avc_bitstream;
 
+#ifndef __BSD_VISIBLE
```

Comment:
I think that it's safer to change that to:

````
#ifndef __OpenBSD__
````

...or properly check for `swap32()` existence in an OS-agnostic way.

(From a quick skim both DragonFlyBSD and FreeBSD defines `__BSD_VISIBLE` but none of them has a `swap32(3)` function and if that's applied I think that would break `intel-vaapi-driver` there.)